app/controllers/facility_reviews_controller.rb

```diff
@@ -61,6 +62,26 @@ def create_success
 
   private
 
+  def filter_matching_space_types
```

Comment:
These two functions are never called in `new`, and so are unused.

Reply:
Ah I probably extracted those to functions without testing (probably because of RuboCop 😆)

Reply:
Rubocop strikes again!